assets/components/LogViewer/CopyLogMessage.vue

```diff
@@ -19,17 +19,21 @@ const { message } = defineProps<{
 }>();
 
 const { showToast } = useToast();
+const { copy, isSupported, copied } = useClipboard();
+const { t } = useI18n();
 
 function copyLogMessageToClipBoard() {
-  navigator.clipboard.writeText(message);
-
-  showToast(
-    {
-      title: "Copied",
-      message: "Log message copied to clipboard",
-      type: "info",
-    },
-    { expire: 2000 },
-  );
+  copy(message).then(() => {
```

Comment:
Will update this file in a moment, with error handled

Reply:
I think this is good to go!

Reply:
Use `await` here. :) You should rarely need `.then()`.